apps/shared/src/components/canvas/components/reactflow-overrides.css

```
@@ -0,0 +1,298 @@
+	margin: 0.1rem 0;
+}
+.rr-annotation-content blockquote {
+	border-left: 3px solid var(--rr-text-disabled);
+	padding-left: 0.5rem;
+	margin: 0.25rem 0;
+	font-style: italic;
+	color: var(--rr-text-secondary);
+}
+.rr-annotation-content code {
+	font-size: 0.7rem;
+	font-family: 'Monaco', 'Menlo', 'Consolas', monospace;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Monaco" (font-family-name-quotes)

(font-family-name-quotes)